### PR TITLE
feat: article extender question/answer models + service

### PR DESCRIPTION
**Pull request by bobaaaaa**
## What

- added a new `Body` type `ARTICLE_EXTENDER`
- `BodyNode` type is just a `string` -- using `question` for filtering
- add new `ArticleExtenderServer` method/function to receive answers for a question
- deprecated `TRUST_BOX` (already deleted in cms, adapter, frontend)

## Example `Body` with initial questions

```json
{
  "children": [
    {
      "children": [],
      "elements": [],
      "fields": {},
      "type": "question",
      "text": "Was hilft gegen Ungeziefer im Garten?",
      "reference": null
    },
  ],
  "type": "ARTICLE_EXTENDER"
}
```

### CODE REVIEW

**Review by maxfrauscher — DISMISSED** [review]
ich finds sauber! 🚀

**Review by harryherbig — APPROVED** [review]
LGTM

File: stroeer/page/article/v1/article_extender_service.proto [hunk 1]
```diff
@@ -33,11 +36,34 @@ service ArticleExtenderService {
...
+ * | `question`       | `string`            | [required] The question for the returned answers. |
+ *
+ * @CodeBlockStart protobuf
+ */
+message GetAnswerRequest {
+  string question = 1;
+}
+
+message GetAnswerResponse {
+  string answer_text = 1;
```
Comment (thisismana) [inline stroeer/page/article/v1/article_extender_service.proto:66]: Within this message, on the other hand, we can be generic ... stripping the prefix?

- `AnswerResponse.answer: string`
- `AnswerResponse.sources: list[Article]`

File: stroeer/page/article/v1/article_extender_service.proto [hunk 2]
```diff
@@ -33,11 +36,34 @@ service ArticleExtenderService {
...
+  repeated stroeer.core.v1.Article answer_sources = 2;
+  repeated string next_questions = 3;
```
Comment (thisismana) [inline stroeer/page/article/v1/article_extender_service.proto:68]: _Further questions_ would be the correct term.

**Review by thisismana — APPROVED** [review]
looks good, just left a few naming suggestions

**Pull request merged**